src/components/lib/ConnectWalletDialog.js

```diff
@@ -130,6 +187,9 @@ export const ConnectWalletDialog = ({ shown, onConnected }) => {
 	return <div>
 		<div className={`modal connect-wallet ${shown ? 'open' : 'closed'}`}>
 			<div className="modal-body">
+				<div className="close">
+					<div className="x" onClick={onClose}>&#9587;</div>
+				</div>
```

Comment:
I would rather prefer a boutonnière, my dear sir.

Reply:
I'll chuck that in another PR - we've still got the lingering global styles on `<button>`.